### PR TITLE
refactor(docs): remove duplications from main style

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: apps/docs/_layouts/base.njk [hunk 1]
```diff
@@ -99,7 +99,7 @@
     <dark-mode-toggle	slot="action-items" appearance="toggle" permanent></dark-mode-toggle>
     <vwc-nav-item slot="action-items" icon="github-mono" href="https://github.com/vonage/vivid-3" aria-label="View on GitHub" role="navigation"></vwc-nav-item>
 
-  <vwc-side-drawer id="sidedrawer" open slot="app-content">
+  <vwc-side-drawer open slot="app-content">
```
Comment (rachelbt) [inline apps/docs/_layouts/base.njk:102]:
```suggestion
  <vwc-side-drawer open id="sidedrawer"  slot="app-content">
```

**Review by rachelbt — CHANGES_REQUESTED** [review]

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -307,13 +156,6 @@ article > .cbd-container {
...
-.heading-wrapper {
-  position: relative;
-  padding-inline-start: 20px;
-  margin-inline-start: -20px;
-
-}
-
 article > h1 {
   padding-inline: 16px;
   padding-block-start: 40px;
```
Comment (rachelbt) [inline apps/docs/assets/styles/main.css:161]:
```suggestion
  padding-block-start: 20px;
```
there's too much extra space at the top

Comment (yinonov) [reply apps/docs/assets/styles/main.css:161]: unrelated to this scope

File: apps/docs/_layouts/base.njk [hunk 3]
```diff
@@ -99,7 +99,7 @@
     <dark-mode-toggle	slot="action-items" appearance="toggle" permanent></dark-mode-toggle>
     <vwc-nav-item slot="action-items" icon="github-mono" href="https://github.com/vonage/vivid-3" aria-label="View on GitHub" role="navigation"></vwc-nav-item>
 
-  <vwc-side-drawer id="sidedrawer" open slot="app-content">
+  <vwc-side-drawer open id="sidedrawer"  slot="app-content">
```
Comment (vng-vivid) [inline apps/docs/_layouts/base.njk:102]:
```suggestion
  <vwc-side-drawer id="sidedrawer" open slot="app-content">
```

File: apps/docs/_layouts/base.njk [hunk 4]
```diff
@@ -88,7 +88,7 @@
   </head>
 
 <body>
-<vwc-header id="header-main" slot="app-content">
+<vwc-header id="header-main"  slot="app-content">
```
Comment (vng-vivid) [inline apps/docs/_layouts/base.njk:91]:
```suggestion
<vwc-header id="header-main" slot="app-content">
```

**Review by vng-vivid — CHANGES_REQUESTED** [review]

**Review by vng-vivid — APPROVED** [review]

**Pull request merged**